var/spack/repos/builtin/packages/pnmpi/package.py

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+from spack import *
+
+
+class Pnmpi(CMakePackage):
+    """PnMPI is a dynamic MPI tool infrastructure that builds on top of
+       the standardized PMPI interface. """
+
+    homepage = "https://github.com/LLNL/PnMPI"
+    url      = "https://github.com/LLNL/PnMPI/releases/download/v1.7/PnMPI-v1.7-full.tar.gz"
+
+    version('1.7', '8040c1558c0deaa3d964c35d1760f3a8')
+
+    variant('fortran', default=True,
+            description='Configure PnMPI with Fortran support')
+    variant('tests', default=True,
```

Comment:
I would set the default to `False` as long as the tests are not run (and they are not installed usually as well).

Otherwise, spack will try to run `make test` after the build which needs to work in such a case (does it? how large are the tests?)

Reply:
`make test` does seem to work. The entire build takes around 45 seconds on my MacBook Pro 2017, and as far as I could tell, the tests ran. I've set the default to `False` anyway.